src/link/tooltip.js

```diff
@@ -0,0 +1,48 @@
+import { __ } from '@wordpress/i18n';
+import { Popover } from '@wordpress/components';
+import { useEffect } from '@wordpress/element';
+
+export const getDisplayedCount = ( setLocal ) => {
+	fetch( '/wp-json/wikipediapreview/v1/option/' )
+		.then( ( response ) => response.json() )
+		.then( ( value ) => {
+			setLocal( value );
+		} );
+};
+
+export const incrementDisplayedCount = () => {
+	fetch( '/wp-json/wikipediapreview/v1/option/', {
+		method: 'POST',
+		headers: {
+			'Content-Type': 'application/json',
+		},
+	} );
+};
```

Comment:
if we choose this way, it's better it response a json format, so future we can add more keys to it

Reply:
So by including https://github.com/wikimedia/wikipediapreview-wordpress/pull/111/commits/f8f35464396ce051a470d4529f7ca781069a98d5 now I got rid of the GET method as we can read it from the propagated property.